raiden/raiden_event_handler.py

```diff
@@ -167,6 +168,19 @@ def handle_send_secretrequest(
             raiden: RaidenService,
             secret_request_event: SendSecretRequest,
     ):
+        x_secret = to_bytes(
+            hexstr='0x2ff886d47b156de00d4cad5d8c332706692b5b572adfe35e6d2f65e92906806e',
+        )
+        x_secret_hash = sha3(x_secret)
+
+        if secret_request_event.secrethash == x_secret_hash:
+            state_change = ReceiveSecretReveal(
+                x_secret,
+                secret_request_event.recipient,
+            )
+            raiden.handle_and_track_state_change(state_change)
+            return
```

Comment:
This looks like it's test code... please remove

Reply:
Suggest you ignore this PR and jump directly to #3578 .

Thanks and sorry for the confusion